build.py

```diff
@@ -88,6 +88,13 @@ def build():
         with zipfile.ZipFile(wheel_path) as wheel:
             wheel.extract(libparsec_path, path=BASEDIR)
 
+            if build_profile == "ci" and sys.platform == "linux":
```

Comment:
@Volham22 @FirelightFlagboy Please add comment, and have the reflex to wonder about comments in when reviewing a PR 🙏 

This is a typical piece of code that will require efforts to understand why it is needed (why only on linux ? why only on ci ? what `parsec.libs` is for ? what is generated by ? answering those questions means ones has to do tests by building on his machine and on the CI)

Reply:
Besides I find is really weird to only have this check in CI mode (it seems like a very ad-hoc change, why don't we have the same issue in dev mode while maturin basically does the same thing for ci and dev... so I would guess it is an issue with `poetry install -E backend -E core` and not `maturin develop`)